packages/stores/types.ts

```diff
@@ -244,16 +249,30 @@ type TNotificationStore = {
 
 type TTradersHubStore = {
     closeModal: () => void;
-    content_flag: any;
-    openModal: (modal_id: string, props?: any) => void;
+    content_flag: 'low_risk_cr_eu' | 'low_risk_cr_non_eu' | 'high_risk_cr' | 'cr_demo' | 'eu_demo' | 'eu_real' | '';
+    combined_cfd_mt5_accounts: DetailsOfEachMT5Loginid &
+        {
+            short_code_and_region: string;
+            login: string;
+            sub_title: string;
+            icon: 'Derived' | 'Financial' | 'Options' | 'CFDs';
+        }[];
+    openModal: (modal_id: string, props?: unknown) => void;
+    selected_account: {
+        login: string;
+        account_id: string;
+    };
 };
 
-export type TRootStore = {
+/**
+ * This is the type that contains all the `core` package stores
+ */
+export type TCoreStores = {
     client: TClientStore;
     common: TCommonStore;
     menu: TMenuStore;
     ui: TUiStore;
-    modules: Record<string, any>;
+    modules: Record<string, Record<string, unknown>>;
```

Comment:
```suggestion
    modules: Record<string, any>;
```
@hamid-deriv Let's keep it as it was, Otherwise we will need to do `as unknown as FooStore` in other places.